src/main/scala/example/models/Book.scala

```diff
@@ -5,7 +5,10 @@ import scala.util.{Try, Success, Failure}
 import example.models.errors.{TransformError}
 import example.models._
 
-case class Book(isbn: String)
+case class Book(isbn: String) {
+  def linkToGoodreads: String = s"https://www.goodreads.com/book/isbn/$isbn"
+}
```

Comment:
My comment still stands from the other pr on the extensibility factor, but not a blocker for this PR.